lib/membrane_rtc_engine/endpoints/webrtc_endpoint.ex

```diff
@@ -87,6 +87,10 @@ if Code.ensure_loaded?(Membrane.WebRTC.EndpointBin) do
                   default: false,
                   description: "Set to `true`, to use integrated TURN instead of libnice"
                 ],
+                integrated_turn_domain: [
+                  spec: binary() | nil,
+                  description: "Domain address of integrated TURN Servers. Required for TLS TURN"
+                ],
```

Comment:
What about putting this in `integrated_turn_options`?

Reply:
`integrated_turn_options` is part of `Membrane.ICE.Endpoint` API. `integrated_turn_domain` is not used by `Membrane.ICE.Endpoint` or any of it's dependencies, so I decided to not put it there